01-report.js

```diff
@@ -11,15 +11,17 @@ export function getLast12MonthReport(database) {
     // Please don't get stuck too long on this one, ask for help if you need it.
     // Also, reading the doc might be helpful :D
     const today = new Date();
-    const d = new Date();
-    const year = d.getFullYear();
+    let date = new Date(); 
```

Comment:
Even if the value changes with `setFullYear` and `setMonth`, the `date` variable is never reassigned (you never have `date = something`, so it can be `const`. It is quite counterintuitive, I agree, but a `const` object may change.